datafusion/physical-optimizer/tests/enforce_sorting.rs

```diff
@@ -1,861 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This was consolidated with the exsiting `enforce_sorting.rs` tests (same for the others below)